_notices/rsn0017.md

```diff
@@ -9,8 +9,8 @@ notice_id: 17 # should match notice number
 notice_pin: true # set to true to pin to notice page
```

Comment:
Should we unpin this one and pin the following? We have a lot of RSNs pinned, not sure we need them all.

Reply:
I agree with this